.github/workflows/oracle_pg_regression.yml

```
@@ -39,9 +39,10 @@ jobs:
             --with-ossp-uuid  --with-libxml --with-libxslt  --with-perl \
             --with-icu
     - name: configure - macos
-      if: ${{ matrix.os == 'macos-12' }}
+      if: ${{ matrix.os == 'macos-latest' }}
       run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2155:warning:2:8: Declare and assign separately to avoid masking return values

(shellcheck)